src/events/components/SearchModule/DateRangeInput.tsx

```diff
@@ -0,0 +1,109 @@
+import { DateTime } from 'luxon';
+import React, { FC, useRef } from 'react';
+import { DayPickerProps } from 'react-day-picker';
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+import 'react-day-picker/lib/style.css';
+
+import style from './search.less';
+
+const WEEKDAYS_SHORT = ['Sø', 'Ma', 'Ti', 'On', 'To', 'Fr', 'Lø'];
+const MONTHS = [
+  'Januar',
+  'Februar',
+  'Mars',
+  'April',
+  'Mai',
+  'Juni',
+  'Juli',
+  'August',
+  'September',
+  'Oktober',
+  'November',
+  'Desember',
+];
+
+const WEEKDAYS_LONG = ['Søndag', 'Mandag', 'Tirsdag', 'Onsdag', 'Torsdag', 'Fredag', 'Lørdag'];
+
+const FIRST_DAY_OF_WEEK = 1;
+
+const LABELS = { nextMonth: 'Neste måned', previousMonth: 'Forrige måned' };
```

Comment:
Just as a FYI, I added this manually, so that we don't have to add `Moment.js` as a dependency, because it is unnecessary imo

Reply:
+1 this, looked at the documentation I linked and fully agree with you